pkg/kubelet/cm/topologymanager/topology_manager.go

```diff
@@ -223,15 +223,22 @@ func (m *manager) RemoveContainer(containerID string) error {
 }
 
 func (m *manager) Admit(attrs *lifecycle.PodAdmitAttributes) lifecycle.PodAdmitResult {
-	// Unconditionally admit the pod if we are running with the 'none' policy.
-	if m.policy.Name() == PolicyNone {
-		return lifecycle.PodAdmitResult{Admit: true}
-	}
-
 	klog.Infof("[topologymanager] Topology Admit Handler")
 	pod := attrs.Pod
 
 	for _, container := range append(pod.Spec.InitContainers, pod.Spec.Containers...) {
+		if m.policy.Name() == PolicyNone {
+			err := m.allocateAlignedResources(pod, &container)
```

Comment:
Why do I have to call allocate with none?

Is it cpu manager on but topology manager off?

Reply:
Yes, cpu manager static policy and topology manager none policy

Reply:
thanks for clarification , fix makes sense as a result